presto-main/src/test/java/com/facebook/presto/cost/TestCostCalculator.java

```diff
@@ -136,16 +138,16 @@ public void testRepartitionedJoin()
                 costs,
                 stats)
                 .cpu(12000 + 6000 + 1000 + 6000 + 1000)
-                .memory(1000)
-                .network(0);
+                .memoryUnknown()
+                .networkUnknown();
 
         assertCostEstimatedExchanges(
                 join,
                 costs,
                 stats)
                 .cpu(12000 + 6000 + 1000 + 6000 + 1000 + 6000 + 1000)
-                .memory(1000)
-                .network(6000 + 1000);
+                .memoryUnknown()
```

Comment:
I agree with changing `memory(0)` with `memoryUnknown()` in this patch, but why does this change?
Same comment below to similar cases with both `memory` and `network`.